keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/PersistentEvent.kt

```diff
@@ -0,0 +1,21 @@
+package com.netflix.spinnaker.keel.events
+
+import java.time.Clock
+import java.time.Instant
+
+abstract class PersistentEvent {
```

Comment:
This is the new base class for all persistent events. It defines the common properties I expect to see in all such events regardless of the sub-class.

Reply:
Again I find the name to be a bit confusing... maybe it's just me :) but can't it be just `event`? curios to hear other opinions.